packages/suite/src/utils/suite/device.ts

```diff
@@ -121,6 +121,12 @@ export const getFwVersion = (device: AcquiredDevice) => {
     return `${features.major_version}.${features.minor_version}.${features.patch_version}`;
 };
 
+export const getFwRevision = (device: AcquiredDevice) => {
```

Comment:
```suggestion
export const getFwRevision = (device: AcquiredDevice) => Buffer.from(device.features.revision, 'hex').toString();
```
i would prefer this one liner.
looks cleaner imo and doesnt assign any variables

Reply:
on second thought, you don't need to create utility for that since it's used only in 1 place.
it could be used/assigned directly in settings/device view

Reply:
14d766f

Reply:
i'm still not convinced about not using the Buffer, but i leave it for @keraf to decide, he is the pipeline master :)